src/components/BookmarkContainer.jsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/jsx-indent */
+/* eslint-disable react/jsx-closing-tag-location */
+/* eslint-disable react/no-array-index-key */
+/* eslint-disable react/jsx-wrap-multilines */
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+import BookmarkItem from '../views/BookmarkItem';
+import { getBookmarkedArticles } from '../actions/getBookmark';
+import Pagination from './Pagination';
+import * as selectors from '../selectors/getBookmarkSelector';
+
+/**
+ * @description - returns all articles bookmarked by a user
+ * @return {JSX}
+ */
+function BookmarkContainer(props) {
+  const [viewType, setViewType] = useState('row view-row');
+
+  const { articles, getBookmarkedArticles } = props;
+
+  const getArticlesHandler = () => {
+    getBookmarkedArticles();
+  };
+
+  const changeViewToHorizontal = () => {
+    setViewType('row view-row list-view');
+  };
+
+  const changeViewToVertical = () => {
+    setViewType('row view-row');
+  };
+
+  useEffect(() => (
+    getArticlesHandler()
+  ), []);
+
+  return (
+    <>
+      <header className="page-header" data-pg-collapsed>
+        <h1>Bookmarks </h1>
+        <div className="btn-group btn-group-toggle float-right view-changer" data-toggle="buttons">
+          <label className="btn active" onClick={changeViewToVertical}>
+            <input type="radio" name="options" autoComplete="off" checked value="" />
+            <i className="fas fa-th" />
+          </label>
+          <label className="btn" onClick={changeViewToHorizontal}>
+            <input type="radio" name="options" autoComplete="off" value="list-view" />
+            <i className="fas fa-list" />
+          </label>
+        </div>
+      </header>
+      {
+        articles && articles.length !== 0
+          ? (
+            <>
+              <div className={viewType}>
+                {
+                  // eslint-disable-next-line arrow-body-style
+                  articles.map((article, indx) => {
+                    return (
+                      <BookmarkItem
+                        key={indx}
```

Comment:
We should use the article Id as the key instead as recommended by the React team and enforced in our Eslint configuration. What do you think?

Reply:
@valenstical fixed